client/src/main/java/tds/config/ClientSystemFlag.java

```diff
@@ -6,10 +6,12 @@
  Represents a record in the {@code configs.client_systemflags} table.
  */
 public class ClientSystemFlag {
+    public static final String ANONYMOUS_STUDENT_AUDIT_OBJECT = "AnonymousTestee";
```

Comment:
I am making these string values that have meaning public to help at least indicate what some of the valid types are since this is relied on by the exam microservice.